app/assets/javascripts/client/inventories/MessageElementCtrl.js

```diff
@@ -6,12 +6,13 @@
 
   function MessageElementCtrl() {
     var vm = this;
-    vm.messageTitle = function(category) {
+    vm.messageTitle = function(category, kind) {
+      var subjectKind = kind || 'Inventory';
```

Comment:
should I change the way we're passing the category in too? we're doing the same thing for the category, it's actually `message.category` that I can definitely get via `$scope.message.category`. it's no pain.